_source/user-guide/users/single-sign-on/index.md

```diff
@@ -12,16 +12,16 @@ contributors:
   - hidan
 ---
 
-Single sign-on (SSO) allows you to manage access to your Logz.io account
+Single sign-on (SSO) lets you manage access to your Logz.io account
 from a single user and identity management tool.
 
 ## What is SSO?
 
 SSO is a way to authenticate and sign in to different services
 with the same credentials.
-With SSO, users won't need to remember
+With SSO, users don't need to remember
 different usernames and passwords for each service,
-and you can manage access to all your services from one service.
+and you can manage access to all your services from one source.
```

Comment:
Do you mean "data source"?

Reply:
I meant that they can use one service to handle everything. However, I did use the word service three times in that sentence, and source can also work, so I'm good with this fix.